main.go

```diff
@@ -33,8 +33,10 @@ import (
 	corev1 "k8s.io/api/core/v1"
 
 	extensionswebhook "github.com/gardener/gardener/extensions/pkg/webhook"
+
 	kupidv1alpha1 "github.com/gardener/kupid/api/v1alpha1"
 	"github.com/gardener/kupid/pkg/webhook"
+	kupidWebhook "github.com/gardener/kupid/pkg/webhook"
```

Comment:
```suggestion
	kupidwebhook "github.com/gardener/kupid/pkg/webhook"
```

Package names (even renamed ones) should not use capitalised letters.

Reply:
Thanks addressed with 838c1e960a0562913b883e2649d9bfd8fd18b23f